### ISSUE TITLE
Network - Click event only gives edgeId's in params.items

### ISSUE THREAD

**Issue by stonerich** [almende/vis#3871]
Please make sure to **read the following list** before creating a new issue:

I get this when using the https://cdnjs.cloudflare.com/ajax/libs/vis/4.21.0/vis.min.js distribution.

Jsbin:
http://jsbin.com/jemavow/2/edit?html,output

The documentation says that the items array will contain both nodeId's and edgeId's. However, it seems to only include edges even when clicking on a node. When clicking on a node the edgeIds seems to be the ondes that would pass "into" the node at the clickpoint.

For example, clicking on node 2 in the example can give from zero up to 3 edgeId in the result but node 2 itself is never part of the items.

Using the this.getNodeAt(params.pointer.DOM) methods does give the correct node (and I'm aware that I could use the selectNode instead) but at least the documentation seems a bit misleading.

I assume that the problem is in SelectionHandler.js, getClickedItems but its not immediately obvious whet the problem is.

**Comment by Menighin** [comment]
Ran into the same issue. A workaround is using `getNodeAt()` with the `pointer` position.

**Comment by brichbe** [comment]
@stonerich is this issue still relevant?  Using Chrome, your Jsbin works correctly for me.  When I click a node, I see that node's ID in the console log output and within your HTML output area (in both places, you're getting the clicked node ID from params.nodes).

**Comment by Menighin** [comment]
Hi @stonerich, I think you misunderstood the issue. 
The `params.nodes` is showing the node ID correctly. The thing is that the documentation says (in the Events part) that the `params.items` would also contain both ID's for nodes and edges, but it is showing only edges IDs (sometimes not even the edges IDs)

**Comment by brichbe** [comment]
You're right -- I see that the `params.items` contains wrong/misleading values.  I wonder what was the original motivation for adding `params.items`, since it seems redundant with `params.nodes` and `params.edges`. In fact, even the documentation for `params.items` is strangely worded -- the wording suggests a `labelId` may be included, which is weird because labels don't have Ids, labels are attribute String values part of the node/edge data structures. 

Either way, agree this is a problem that needs to be fixed (in my opinion best by removing/deprecating support for `params.items`)
